setup.cfg

```diff
@@ -0,0 +1,2 @@
+[codespell]
+skip = pyscriptjs/node_modules/*,*.js,*.json
```

Comment:
If we are going to have a `setup.cfg`, we may as well place the `flake8` settings here as well: https://flake8.pycqa.org/en/latest/user/configuration.html

Reply:
Fixed but... I only use it for codespell because yaml is so baffled by the * character.  In general, I hate spreading configuration state across multiple files.